### PR TITLE
Feature/new feature

### PR DESCRIPTION
**Pull request by wojcharuza**

### CODE REVIEW

File: NewFeature [hunk 1]
```diff
@@ -0,0 +1,8 @@
+NewFeature
+Adf
+Asdfggf
+Sdfg
+Fix fix fix
+Asdsdf
```
Comment (FeniXb3) [inline NewFeature:6]: I thnk this name is not clear. Can you create a better one?

File: NewFeature [hunk 2]
```diff
@@ -0,0 +1,8 @@
+NewFeature
+Adf
+Asdfggf
+Sdfg
+Fix fix fix
```
Comment (FeniXb3) [inline NewFeature]:
```suggestion
Fix fix bugfix
```

**Review by FeniXb3 — CHANGES_REQUESTED** [review]

File: NewFeature (hunk 1 above)
Comment (wojcharuza) [reply NewFeature:6]: no, thanks :)

Comment (FeniXb3) [reply NewFeature:6]: Ok, fine

**Review by FeniXb3 — COMMENTED** [review]
Simple comment

**Review by FeniXb3 — APPROVED** [review]
Great job

**Pull request merged**